adoc/retail_migr_chap_prepare.adoc

```diff
@@ -26,6 +25,28 @@ You will migrate the existing server installation to new hardware or virtual mac
 This document contains several different migration scenarios.
 Ensure you pick the most appropriate migration scenario for your environment.
 
-If you are migrating from {slepos} 11 to {smr}, and you do not have a {susemgr} 3.2 installation, see xref:retail_migr_chap_slepostosuma.adoc#retail_migr.chap.slepostosuma[SLEPOS to new SUSE Manager].
+If you are migrating from {slepos} 11 to {smr}, and you do not have a {susemgr} installation, see xref:retail_migr_chap_slepostosuma.adoc#retail_migr.chap.slepostosuma[SLEPOS to new SUSE Manager].
 
-If you are migrating from {slepos} 11 to {smr}, and you already have a {susemgr} 3.2 installation that you want to use, see xref:retail_migr_chap_slepostosumaind.adoc#retail_migr.sect.slepostosuma.s2a[SLEPOS to Independent SUSE Manager].
+If you are migrating from {slepos} 11 to {smr}, and you already have a {susemgr} installation that you want to use, see xref:retail_migr_chap_slepostosumaind.adoc#retail_migr.sect.slepostosuma.s2a[SLEPOS to Independent SUSE Manager].
+
+
+
+=== Migrating from {slepos}
+
+{slepos} cannot be upgraded directly to {smr}, so the migration process requires you to perform some manual configuration.
+To assist you in the migration, as much information as possible about the existing hardware configuration and network infrastructure is recorded and then prepared and used for rebuilding the new {smr} installation.
+
+In some cases, this will require a lengthy downtime to perform the migration.
+If you are not able to manage downtime, you can install new servers and run them in parallel to the existing ones while you perform the migration.
+This is especially relevant for large installations.
+
+It is possible to run a SLEPOS Admin server and a {susemgr} server in parallel, as branches that have been migrated will run on the {susemgr} server, while those which have not
+yet been migrated can continue to run on the Admin server.
+This includes all operations, such as adding new terminals, or building and deploying new images.
+
+However, if you run network services (especially DHCP) on the Branch servers, you will not be able to run both old and new branch servers in parallel, because they can conflict with each other.
```

Comment:
you will not be able to run both old and new branch servers in the same branch in parallel,